spec/instant.html

```diff
@@ -521,27 +521,15 @@ <h1>ToTemporalInstant ( _item_ )</h1>
           1. NOTE: This use of ToPrimitive allows Instant-like objects to be converted.
           1. Set _item_ to ? ToPrimitive(_item_, ~string~).
         1. If _item_ is not a String, throw a *TypeError* exception.
-        1. Let _epochNanoseconds_ be ? ParseTemporalInstant(_item_).
+        1. Let _parsed_ be ? ParseTemporalInstantString(_item_).
+        1. If _parsed_.[[Z]] is *true*, let _offsetNanoseconds_ be 0.
+        1. Else, let _offsetNanoseconds_ be ! ParseDateTimeUTCOffset(_parsed_.[[OffsetString]]).
```

Comment:
I'm not a fan of an Else step after a single-line If step, but that can be cleaned up later if you don't want to change it now.

Reply:
OK, I'll replace it with `; otherwise...`